go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/kontena/kubelet-rubber-stamp
+
+go 1.11
+
+require (
+	github.com/operator-framework/operator-sdk v0.4.0
+	k8s.io/api v0.0.0-20190918155943-95b840bb6a1f
+	k8s.io/apimachinery v0.0.0-20190913080033-27d36303b655
+	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
```

Comment:
Just curious, how did you decipher which "version" to take in to "match" the previously used versions?

Reply:
Frankly, I can't recall it. I played with `go build` / `go mod`. Additionally made several search, but my proposal is to update it to the latest which has more deterministic tagging - fortunately they [simplified version tagging](https://github.com/kubernetes/client-go/blob/master/INSTALL.md#add-client-go-as-a-dependency). I can make a PR on top of this PR to update dependencies if it is okay for you.